target/ppc/insn64.decode

```diff
@@ -54,6 +54,11 @@
                 ...... ..... ..... ..... ..... .. .... \
                 &8RR_XX4 xt=%8rr_xx_xt xa=%8rr_xx_xa xb=%8rr_xx_xb xc=%8rr_xx_xc
 
+&XX4_uim3       xt xa xb xc uim3
+@XX4_uim3       ...... .. .... -- ............... uim3:3 \
```

Comment:
I suggest using this just for defining the fields (either fields or '.'s), and leaving '-'s to the instruction pattern (XXPERMX)